sdk/core/Azure.Core.TestFramework/src/OperationInterceptor.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using Castle.DynamicProxy;
+
+namespace Azure.Core.TestFramework
+{
+    internal class OperationInterceptor : IInterceptor
+    {
+        private static readonly object NoWaitDelay = TimeSpan.FromMilliseconds(1);
```

Comment:
Is this just to make sure they run async? While I haven't gotten down to the usage yet, I know that in KV and Search we use `TimeSpan.Zero` for playback and it works fine. That said, a 1ms delay is of no concern. Just curious.

Reply:
Yeah, that was the original intent but I changed it to `TimeSpan.Zero`